### PR TITLE
feature: search pages interface

### PR DESCRIPTION
**Pull request by manzcode**
The interface on Figma is a bit incomplete for the widescreen component, so I supplemented it. Check it.

### CODE REVIEW

File: client/src/pages/Search.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import avatar from "../assets/dashboard/avatar.svg";
+import { FaSearch } from "react-icons/fa";
+import SearchProfileCard from "../components/SearchProfileCard";
+
+function Search() {
+  return (
+    <main className="lg:text-xl lg:text-lg">
```
Comment (marktlinn) [inline client/src/pages/Search.jsx]: I think we could delete one of these rules?

File: client/src/pages/Search.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        <label
+          htmlFor="search"
+          className="hidden lg:flex lg:justify-center lg:items-center lg:text-xl text-primary font-bold text-sm col-span-4"
+        >
+          The Search Start here
+        </label>
+        <form action="#" className="my-4 col-span-7 w-full">
+          <div className="flex">
+            <input
+              className="border-solid rounded-tl-lg p-2 rounded-bl-lg text-sm w-full"
```
Comment (marktlinn) [inline client/src/pages/Search.jsx]: On my browser the border doesn't show up, I think it has to be just "border" in Tailwind and it should show up.

**Review by marktlinn — CHANGES_REQUESTED** [review]
It looks really good Mario, look at the comments I put and let me know what you think 🙂

File: client/src/pages/Search.jsx (hunk 1 above)
Comment (manzcode) [reply client/src/pages/Search.jsx]: 😲 ok

File: client/src/pages/Search.jsx (hunk 2 above)
Comment (manzcode) [reply client/src/pages/Search.jsx]: yes, I'll add it.

**Review by manzcode — COMMENTED** [review]
I just add border to the input and delete the duplicate text size on large.

**Review by marktlinn — APPROVED** [review]
Looks good to me! 🙂

**Pull request merged**